File: go/1043.go
// Funcional no beecrowd
package main

import (
	"fmt"
	"math"
)

func main() {
	var A float64
	var B float64
	var C float64

	fmt.Scan(&A)
	fmt.Scan(&B)
	fmt.Scan(&C)

	ba := math.Abs(B - C)

	if ba < A && A < (B+C) && (A-C) < B && B < (A+C) && (A-B) < C && C < (A+B) {
		fmt.Printf("Perimetro = %.1f\n", (A + B + C))
	} else {
		fmt.Printf("Area = %.1f\n", (((A + B) / 2) * C))
	}
}

File: Lua/Iniciante/1043.lua
-- Funcional no beecrowd!
local A = io.read("n")
local B = io.read("n")
local C = io.read("n")

local AB = A + B
local BC = B + C
local AC = A + C

if math.abs(B - C) < A and A < BC and (A - C) < B and B < AC and (A - B) < C and C < AB then
    print(string.format("Perimetro = %.1f", (A + B + C)))
else
    print(string.format("Area = %.1f", (AB / 2) * C))
end